src/main/java/wooteco/subway/path/domain/SectionEdges.java

```diff
@@ -0,0 +1,25 @@
+package wooteco.subway.path.domain;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import wooteco.subway.line.domain.Line;
+
+public class SectionEdges {
+
+    private final List<SectionEdge> sectionEdgeGroup;
+
+    public SectionEdges(List<SectionEdge> sectionEdgeGroup) {
+        this.sectionEdgeGroup = sectionEdgeGroup;
+    }
+
+    public int calculateDistance() {
+        return sectionEdgeGroup.stream().mapToInt(it -> it.getSection().getDistance()).sum();
```

Comment:
디미터의 법칙을 적용 할 수 있을 것 같아요 :) 
```suggestion
        return sectionEdgeGroup.stream().mapToInt(it -> it.getSectionDistance()).sum();
```